src/doc.cls.php

```diff
@@ -59,6 +59,7 @@ public static function notice_htaccess()
 	{
 		echo '<font class="litespeed-warning">' ;
 		echo '⚠️ ' . __( 'This setting will edit the .htaccess file.', 'litespeed-cache' ) ;
+		echo ' <a href="https://docs.litespeedtech.com/lscache/lscwp/toolbox/#edit-htaccess-tab" target="_blank" class="litespeed-learn-more">' . __( 'Learn More', 'litespeed-cache' ) . '</a>'
```

Comment:
please visit the modified pages to make sure it runs as expected.

Reply:
All fixed & tested now on my side. Ready for final review.